src/components/Resize/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, JSX, useMemo, useEffect } from 'react';
+import CodeBlock from '@theme/CodeBlock';
+
+/**
+ * Resize Component - Simple demo page showing off SpringRoll's SafeScaleManager. Referenced in docs/Examples/resize.mdx
+ * @returns JSX.Element
+ */
+export default function ResizeExample(): JSX.Element {
+  const [width, setWidth] = useState(0);
+  const [height, setHeight] = useState(0);
+  const [ratio, setRatio] = useState(0);
+
+  const [manager, setManager] = useState(null);
+  // SafeScaleManager uses the window object which isn't available in Node
+  // so the build process requires it to be dynamically imported to avoid errors
+  useEffect(() => {
+
+    if (typeof window !== 'undefined' && window.speechSynthesis) {
```

Comment:
Should probably just make it a check for window or a check for whatever safescale manager needs

Reply:
Oh wow, I totally overlooked that when duplicating the speech synthesis example. Yeah I'm going to update it so that it only checks for `window`.